grafana/docker-compose.yml

```diff
@@ -9,6 +9,19 @@ services:
       - PUID=1000
       - PGID=1000
       - TZ=America/Phoenix
+      - GF_AUTH_BASIC_ENABLED=true # leave true while configuring OAuth
+      - GF_AUTH_DISABLE_LOGIN_FORM=false # leave false while configuring OAuth
+      - GF_AUTH_OKTA_ENABLED=true
+      - GF_AUTH_OKTA_NAME=Okta
+      - GF_AUTH_OKTA_ALLOW_SIGN_UP=true
+      - GF_AUTH_OKTA_CLIENT_ID=${OKTA_CLIENT_ID}
+      - GF_AUTH_OKTA_CLIENT_SECRET=${OKTA_CLIENT_SECRET}
+      - GF_AUTH_OKTA_SCOPES=${OKTA_SCOPES}
+      - GF_AUTH_OKTA_AUTH_URL=${GODADDY_OKTA_AUTH_URL}
+      - GF_AUTH_OKTA_TOKEN_URL=${GODADD_OKTA_TOKEN_URL}
+      - GF_AUTH_OKTA_API_URL=${GODADDY_OKTA_USERINFO_URL}
+      - GF_AUTH_OKTA_ALLOWED_GROUPS=${ALLOWED_GODADDY_GROUPS}
+      - GF_AUTH_OKTA_ROLE_ATTRIBUTE_PATH=${ROLE_ATTRIBUTE_PATH}
     volumes:
```

Comment:
docker compose has the ability to scale in the compose file now, instead of defining two services you can define it like this. Remove the container_name since using scaling option will handle naming.

services:
  grafana:
    image: grafana/grafana:latest
    ports:
      - "8080-8082:3000"
     deploy:
       replicas: 3